packages/@webex/plugin-meetings/src/meeting/util.ts

```diff
@@ -371,8 +371,9 @@ const MeetingUtil = {
     displayHints.includes(DISPLAY_HINTS.LEAVE_END_MEETING),
 
   canManageBreakout: (displayHints) => displayHints.includes(DISPLAY_HINTS.BREAKOUT_MANAGEMENT),
-  canBroadcastMessageToBreakout: (displayHints) =>
-    displayHints.includes(DISPLAY_HINTS.BROADCAST_MESSAGE_TO_BREAKOUT),
+  canBroadcastMessageToBreakout: (displayHints, policies) =>
+    displayHints.includes(DISPLAY_HINTS.BROADCAST_MESSAGE_TO_BREAKOUT) &&
+    !!policies[SELF_POLICY.SUPPORT_BROADCAST_MESSAGE],
```

Comment:
issue: policies can be undefined. We should either default the argument for policies here or default policies to be an empty object in the meeting constructor